spike/spike.spec.js

```diff
@@ -0,0 +1,72 @@
+const axios = require('axios');
+axios.defaults.adapter = require('axios/lib/adapters/http');
+
+const FormData = require('form-data');
+const fs = require('fs');
+
+
```

Comment:
slightly crap jest test that let me work out how to talk to gotenburg via js before trying to build an endpoint